Include/cpython/code.h

```diff
@@ -90,6 +91,7 @@ typedef uint16_t _Py_CODEUNIT;
     PyObject *co_weakreflist;     /* to support weakrefs to code objects */    \
     void *_co_code;               /* cached co_code object/attribute */        \
     int _co_firsttraceable;       /* index of first traceable instruction */   \
```

Comment:
I missed when this was added, but maybe we could move this member up with the other `int`s to plug the hole in the struct?

Reply:
Not with breaking code that accesses `co_name` and other nearby fields.
In 3.12, yes we can. Although it looks like we might just drop `_co_firsttraceable` https://github.com/python/cpython/pull/93818#issuecomment-1156205232